pgbouncer/values.yaml

```diff
@@ -51,14 +58,16 @@ settings:
   #  WHERE usename = p_usename;
   #  END;
   #  $$ LANGUAGE plpgsql SECURITY DEFINER;
-  auth_query: SELECT * FROM pgbouncer.get_auth($1) 
+  auth_query: SELECT * FROM pgbouncer.get_auth($1)
 
 connectionLimits:
   maxClientConn: 200
   defaultPoolSize: 200
   minPoolSize: 15
   reservePoolSize: 25
   reservePoolTimeout: 5
+  # maxDbConnections: 0  # zero means unlimited
```

Comment:
since you didn't add templating to set `max_db_connections` in _pgbouncer.ini.tpl` could you remove this line?
imo, having this line commented out suggests that if a chart user un-comments it and changes its value, then the rendered pgbouncer.ini (in Secret) would reflect the new value - but it would not.